### ISSUE TITLE
docker tellform not starting due to Mail configuration sl 

### ISSUE THREAD

**Issue by port22** [tellform/tellform#342]
### What's this issue about?

Tellform complains about SSL problems, despite having `MAILER_SMTP_SECURE` disabled.

```
MAILER_EMAIL_ID=polls@mydomain.intra
MAILER_PASSWORD=123
MAILER_FROM=polls@mydomain.intra
MAILER_SMTP_HOST=mailhog.mydomain.intra
MAILER_SMTP_PORT=25
#MAILER_SMTP_SECURE
```

```
docker-compose up tellform

tellform_1  | unable to find logging file configuration
tellform_1  | --
tellform_1  | Environment:			development
tellform_1  | Port:				5000
tellform_1  | Database:			mongodb://mongo/tellform
tellform_1  | --
tellform_1  | Your mail configuration is incorrect: Error: 140204934195008:error:140770FC:SSL routines:SSL23_GET_SERVER_HELLO:unknown protocol:../deps/openssl/openssl/ssl/s23_clnt.c:797:
tellform_1  | 
docker_files_tellform_1 exited with code 255
```

I just wanted to test the software, I don't care about mailing atm. How to make the app just work?

**Comment by Leopere** [comment]
Currently, the app is not recommended for production use.

**Comment by Leopere** [comment]
closed by #344
